src/components/CouponDetails/index.jsx

```diff
@@ -151,37 +158,28 @@ class CouponDetails extends React.Component {
                 </div>
               </div>
               <div className="row mb-3">
-                <div className="filters col-12 col-md-8">
+                <div className="toggles col-12 col-md-8">
                   <div className="row">
                     <div className="col">
-                      <div className="mb-1">Filter:</div>
-                      <Button
-                        className={['btn-sm', 'mr-2']}
-                        buttonType={selectedFilter === 'not-assigned' ? 'primary' : 'outline-primary'}
-                        label="Not Assigned"
-                        onClick={() => this.setSelectedFilter('not-assigned')}
+                      <InputSelect
+                        ref={this.toggleSelectRef}
+                        className={['mt-1']}
+                        name="table-view"
+                        label="Table View:"
+                        value={selectedToggle}
+                        options={[
+                          { label: 'Not Assigned', value: 'not-assigned' },
+                          { label: 'Not Redeemed', value: 'not-redeemed' },
+                        ]}
                         disabled={this.isTableLoading()}
                       />
                       <Button
-                        className={['btn-sm']}
-                        buttonType={selectedFilter === 'not-redeemed' ? 'primary' : 'outline-primary'}
-                        label="Not Redeemed"
-                        onClick={() => this.setSelectedFilter('not-redeemed')}
+                        className={['ml-2']}
+                        buttonType="primary"
+                        label="Go"
+                        onClick={this.handleToggleSelect}
                         disabled={this.isTableLoading()}
                       />
-                      {selectedFilter &&
```

Comment:
@adamstankiewicz about my question above, If we are going to keep 'default to all records', we may need this. If not, you can ignore this.